setup.py

```diff
@@ -149,15 +149,6 @@ def run(self):
             # as part of the MLflow server monitoring add-on
             "prometheus-flask-exporter",
         ],
-        "pipelines": [
-            "scikit-learn>=1.0",
```

Comment:
We may have to make some small changes to support `scikit-learn<1.0` or restrict MLflow 2.0 to `scikit-learn>=1.0`

Reply:
Why are we restricting scikit-learn `>=1.0` in MLP?

Reply:
Is this because of `get_feature_names_out`?

https://github.com/mlflow/mlflow/blob/f65429478255b88a211d7dd262c159b4939b44b2/mlflow/pipelines/steps/transform.py#L28

Reply:
We could run MLP tests with `scikit-learn < 1.0`.

Reply:
Yes, it's because of `get_feature_names_out`.

Reply:
If so, can we just remove `minimum: "1.0"`?

Reply:
Sounds good!

Reply:
@harupy Did we test this manually?

Reply:
@dbczumar I did in https://github.com/mlflow/mlflow/pull/6773.